lib/JointspaceClient.js

```diff
@@ -469,13 +469,18 @@ class JointspaceClient {
         return this.request('audio/volume', 'GET');
     }
 
-    setVolume(volume) {
+    setVolume(volume, mute) {
+        if (typeof mute === 'undefined') {
+            mute = false;
+        }
         return this.request('audio/volume', 'POST', {
             current: volume,
-            muted: false
+            muted: mute
         });
     }
 
+    /* DOES not work on 804 - probably requires current to be set even when muting
+    use setVolume function with either static volume or get current first
```

Comment:
Good catch. 

Sending the mute request without a current volume does work on my TV. I will test if your solution also works on my TV otherwise we need to look for another way to fix this. For example sending both mute requests.

Reply:
It should work... I've implemented storing of the previous volume.current, so on unmuting you are returned to you previous volume setting. Not sure why it is not working on 804 though, was playing for some time with this and this was only possible solution I've found